code/modules/client/client procs.dm

```diff
@@ -78,6 +78,9 @@
 			C = M.client
 		if(!C) // Might be a stealthmin ID, so pass it in straight
 			C = href_list["priv_msg"]
+		else if(C.UID() != href_list["priv_msg"])
+			to_chat(src, "<span class='danger'>Error: Private-Message: Client not found.</span>")
```

Comment:
Error 404 client not found 😎👌

Reply:
Please add 404, I beg you

Reply:
Close enough? ;)

Reply:
Lmao yeah, I see that comment. Nicely done.

The opportunity was perfect :rofl: